app/config/publications/overview-table-columns.js

```diff
@@ -12,6 +33,13 @@ export default [
     translationKeySmall: 'publications-table-comment-small',
     showByDefault: true,
   },
+  {
+    keyName: 'pages',
+    translationKey: 'publications-table-pages',
+    translationKeySmall: 'publications-table-pages-small',
+    // no sort: is aggregated in frontend
```

Comment:
Sortering hiervan is onmogelijk. Dit is niet volgens het design.

Reply:
Inderdaad, niet mogelijk. Indien dat een must-have zou blijken voor de gebruikers (wat ik betwijfel) moet er een nieuwe story voorzien worden.